src/main/java/ch/njol/skript/effects/EffScreaming.java

```diff
@@ -0,0 +1,65 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Enderman;
+import org.bukkit.entity.Goat;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Make Entity Scream")
+@Description("Make a goat or enderman (Paper) start or stop screaming.")
```

Comment:
Same as above, I dont feel like the "Paper" part is needed here.
```suggestion
@Description("Make a goat or enderman start or stop screaming.")
```